.travis.yml

```diff
@@ -27,39 +21,6 @@ before_script:
 - echo PATH is $PATH
 - export RUST_BACKTRACE=1
 
-script: ./ci/script.sh
-
-env:
-  matrix:
-    - GIMLI_JOB="test"
-    - GIMLI_JOB="features"
-
-matrix:
-  fast_finish: true
-  include:
-    # Coverage should only run on Linux. tarpaulin currently requires nightly Rust.
-    - rust: nightly
-      os: linux
-      sudo: required
-      env: GIMLI_JOB="coverage"
-    # Building docs only needs to happen on one os and only on stable.
-    - rust: stable
-      os: linux
-      env: GIMLI_JOB="doc"
-    # Benching should only happen on nightly.
-    - rust: nightly
-      env: GIMLI_JOB="bench"
-    # Build a 32 bit target.
-    - rust: stable
-      sudo: required
-      services: docker
-      env:
-        - TARGET=i686-unknown-linux-gnu
-        - GIMLI_JOB=cross
-    # Build a big-endian target.
-    - rust: stable
-      sudo: required
-      services: docker
-      env:
-        - TARGET=powerpc64-unknown-linux-gnu
-        - GIMLI_JOB=cross
+script: |
+  RUSTFLAGS="--cfg procmacro2_semver_exempt" cargo install cargo-tarpaulin
+  cargo tarpaulin --verbose --ciserver travis-ci --coveralls "$TRAVIS_JOB_ID"
```

Comment:
Are we keeping this because of a github actions limitiation?

Reply:
Nope, just because I haven't looked into how to configure `cargo tarpaulin` and github actions together, nor added a token for them to talk to each other.